cluster/saltbase/salt/kube-controller-manager/kube-controller-manager.manifest

```diff
@@ -74,7 +74,7 @@
     "name": "kube-controller-manager",
     "image": "{{pillar['kube_docker_registry']}}/kube-controller-manager:{{pillar['kube-controller-manager_docker_tag']}}",
     "resources": {
-      "limits": {
```

Comment:
What was the original reason for setting `limits`? Looks like we are not setting memory limits though.